src/Microsoft.TestPlatform.Client/DesignMode/DesignModeClient.cs

```diff
@@ -28,6 +28,7 @@ namespace Microsoft.VisualStudio.TestPlatform.Client.DesignMode
     using Microsoft.VisualStudio.TestPlatform.PlatformAbstractions.Interfaces;
 
     using CommunicationUtilitiesResources = CommunicationUtilities.Resources.Resources;
+    using System.Linq.Expressions;
```

Comment:
cannot compile with eyes :) is it needed?

Reply:
Happy to enable the related rule as warning after we merge the `.editorconfig` change if we agree on it.